site/cypress/e2e/Cypress-Feature/office_hours_queue.spec.js

```diff
@@ -3,31 +3,42 @@ import { isPermissionAllowed } from 'cypress-browser-permissions';
 const queueName = 'Cypress Office Hour Queue 1';
 const queueName_random = 'Cypress Office Hour Queue Random';
 const queueName_blank = 'Cypress Office Hour Queue Blank';
-const queueName1 = 'Cypress Office Hour Queue 2';
 const queueCode = 'cypress_test';
 const queueCode1 = 'cypress_test_fail';
 const newQueueCode = 'cypress_update';
 
-const enableQueue = () => {
-    cy.visit(['sample', 'config']); // course setting
-    cy.get('[data-testid="queue-enabled"]').check();
-    cy.get('[data-testid="queue-enabled"]').should('be.checked');
+const checkRows = (rows) => {
+    cy.get('[data-testid="row-label"]')
+        .its('length')
+        .then((length) => {
+            const startingRow = length - rows.length;
+            for (let i = 0; i < rows.length; i++) {
+                cy.get(`[data-testid="student-row-${startingRow + i + 1}"]`).first().as(`row-${i}`);
+                cy.get(`@row-${i}`).find('[data-testid="row-label"]').should('contain', startingRow + i + 1);
```

Comment:
Would it make sense to add 1 to startingRow at line 14 instead of adding one here?